example/src/screens/Room.tsx

```diff
@@ -32,20 +41,47 @@ export const Room = ({ navigation }: Props) => {
   const participants = Membrane.useRoomParticipants();
   const [focusedParticipantData, setFocusedParticipantData] =
     useState<Participant | null>(null);
+  const { applyPrevOrder, orderAndSave, lru } = ParticipantsTracksWindowManager(
+    MAX_NUM_OF_USERS_ON_THE_SCREEN
+  );
 
-  const participantsWithTracks = participants
-    .map((p) => {
-      if (p.tracks.some((t) => t.type === Membrane.TrackType.Video)) {
-        return p.tracks
-          .filter((t) => t.metadata.type !== 'audio')
-          .map((t) => ({
-            participant: p,
-            trackId: t.id,
-          }));
-      }
-      return { participant: p };
-    })
-    .flat();
+  // LRU initialization
+  useEffect(() => {
```

Comment:
shouldn't you put this in the `ParticipantsTracksWindowManager`? it's a hook, it can have `useEffect`